start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java

```diff
@@ -65,7 +65,7 @@ public RootGradleBuildContributor gradleBuildProjectContributor(CustomGradleBuil
 
     @Bean
     public GradleWrapperContributor gradle4WrapperContributor() {
-        return new GradleWrapperContributor("4");
+        return new GradleWrapperContributor("6");
```

Comment:
but the method is still called `gradle4WrapperContributor` ?

Reply:
if you want to upgrade the gradle version that is produced, put that in a different PR ?

Reply:
I opened a separate PR
I will rebase this PR onto the other one once merged

Reply:
rebased onto master